website/static/js/registrationUtils.js

```diff
@@ -779,9 +1211,24 @@ RegistrationManager.prototype.init = function() {
         );
     });
 
-    $.when(getSchemas, getDraftRegistrations).done(function() {
+    var ready = $.when(getSchemas, getDraftRegistrations).done(function() {
         self.loading(false);
     });
+
+    var urlParams = $osf.urlParams();
+    if (urlParams.c && urlParams.c === 'prereg') {
```

Comment:
This may be broken by the recent renaming of the url parameter `c` to `campaign`.